corehq/apps/reports/templates/reports/bootstrap3/partials/base_maps.html

```diff
@@ -0,0 +1,142 @@
+                <div class="detail_data"><%= field.value %></div>
+                <% }); %>
+            </script>
+
+            <script type="text/template" id="legend_size">
+                <table class="sizelegend">
+                    <% _.each(entries, function(e, i) { %>
+                    <tr id="sizerow-<%= i %>">
+                        <td class="circle-container">
+                            <div class="circle"></div>
+                        </td>
+                        <td class="vallabel"><%= e.valfmt %></td>
+                    </tr>
+                    <% }); %>
+                </table>
+            </script>
+
+            <script type="text/template" id="legend_enum">
+                <table class="enumlegend">
+                    <% _.each(enums, function(e, i) { %>
+                    <tr id="enumrow-<%= i %>"><td class="enum"></td><td class="vallabel"><%= e.label %></td></tr>
+                    <% }); %>
+                </table>
+            </script>
+
+            <script type="text/template" id="legend_colorscale">
+                <table class="colorscalelegend">
+                    <tr>
+                        <td id="scalebar"></td>
+                        <td class="tickmarks">
+                            <div class="tickmarks-inner">
+                                <% _.each(ticks, function(e) { %>
+                                <div class="width-spacer"><%= e.label %></div>
+                                <div class="ticklabel" style="top: <%= e.coord %>px;">
+                                    <div class="ticklabel-inner"><%= e.label %></div>
+                                </div>
+                                <% }); %>
+                            </div>
+                        </td>
+                    </tr>
+                </table>
+            </script>
+
+            <script type="text/template" id="metrics-group">
+                <div class="choice" data-bind="if: name() != '_root', click: onclick">
+                    <i class="fa" data-bind="visible: group, css: { 'fa-caret-right': !expanded(), 'fa-caret-down': expanded }"></i>
+                    <span data-bind="text: name, css: { selected: selected }"></span>
+                </div>
+
+                <div data-bind="visible: expanded, foreach: children" class="metric-group">
+                    <div data-bind="template: { name: 'metrics-group' }"></div>
+                </div>
+            </script>
+
+            <div id="results-capped" class="alert alert-info" style="display: none;">
+                <button type="button" class="close" data-dismiss="alert">&times;</button>
+                There are <strong id="rows-total"></strong> results for the chosen report filters.
+                However, map display has been capped at the first <strong id="rows-capped"></strong> results.
+            </div>
+
+            <div>
+                <div id="map" style="width: 100%;"></div>
```

Comment:
verifying that the `width: 100%` is necessary - why doesn't the div default to full width?